Datadog.Trace.sln

```diff
@@ -456,6 +456,19 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Datadog.Trace.Tools.Runner"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Datadog.Trace.Tools.Runner.IntegrationTests", "tracer\test\Datadog.Trace.Tools.Runner.IntegrationTests\Datadog.Trace.Tools.Runner.IntegrationTests.csproj", "{C7EC2FF7-C495-4A35-80DF-1F57B61214AC}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "agent", "agent", "{8132E220-3DD0-41E3-A8CC-E4AC44488BCF}"
+	ProjectSection(SolutionItems) = preProject
+		tracer\test\agent\Directory.Build.props = tracer\test\agent\Directory.Build.props
+	EndProjectSection
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MockAgent", "tracer\test\agent\MockAgent\MockAgent.csproj", "{CA24E8D0-F10A-4E5E-8DF8-9833BBD2C6FD}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TransportsTester", "tracer\test\test-applications\regression\TransportsTester\TransportsTester.csproj", "{B9E183AD-5228-4263-A923-C6E617C4917B}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "samples", "samples", "{D2AC4318-F103-4458-A3A5-56342DE7128E}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Samples.Transport.UnixDomainSocket", "tracer\samples\Samples.Transport.UnixDomainSocket\Samples.Transport.UnixDomainSocket.csproj", "{8EBEF690-1E94-4000-8878-2C66910630B9}"
```

Comment:
Do we still need this sample or does TransportsTester give enough coverage?

Reply:
This sample allows real agent testing with F5, I would say we should keep this sample.